events/events-team/content/meet-and-greet.md

```diff
@@ -20,116 +20,125 @@ Aside from the SIG leads, this event is run by a Meet & Greet Coordinator (MGC),
 
 ## Audience
 
-The audience for the M&G consists of three groups:
+The audience for MKCC consists of three groups:
 
-- SIG and WG Leads and Subproject Owners: anyone who considers themselves a "SIG/WG member" can attend and work the tables.  We particularly want SIG leads and/or Subproject owners who can guide contributors in joining their SIG and/or contributing to a project.
-- New & Casual Contributors: the visitors for the M&G will be people who are interested in contributing to Kubernetes, or have contributed lightly but not really participated in a SIG.
-- Regular Contributors: the M&G is also for regular contributors who are trying to reach out to a SIG, usually about a stuck PR, review, or mutual project.
+- *SIG and WG Leads and Subproject Owners and Committee Members*: anyone who considers themselves a "SIG/WG member" can attend and work the tables. We particularly want SIG leads and/or Subproject owners who can guide contributors in joining their SIG and/or contributing to a project. Committee members are also encouraged to be at a table or otherwise identify themselves with pins to make it easier for contributors to stop by for a chat.
+- *New & Casual Contributors*: the visitors for MKCC will be people who are interested in contributing to Kubernetes or who have contributed lightly but not really participated in a SIG.
+- *Regular Contributors*: MKCC is also for regular contributors who are trying to reach out to a SIG, usually about a stuck PR, review, or mutual project.
 
-Some teams that are not SIGs or WGs may also participate, e.g. ClusterAPI or the CoCC.  There is no specific policy on this, deal with it on a case-by-case basis. Generally speaking, we want to say yes to teams that actually plan to show up as a group. In the rest of this document, "SIGs" should be considered to refer to "SIGs, WGs, and Teams".
+Some teams that are not SIGs or WGs may also participate, e.g. ClusterAPI or the CoCC. There is no specific policy on this, deal with it on a case-by-case basis. Generally speaking, we want to say yes to teams that actually plan to show up as a group. In the rest of this document, "SIGs" should be considered to refer to "SIGs, WGs, and Teams".
 
 ## Time and Location
 
-The M&G is generally held during lunch period on one of the main Kubecon days.  CNCF staff decides the exact schedule in order to not conflict with other lunchtime events.
+MKCC is generally held during lunch period on one of the main Kubecon days. CNCF staff decides the exact schedule in order to not conflict with other lunchtime events.
```

Comment:
If you do an update, might be good to proper-case KubeCon like above. But if not, meh.

Reply:
Agreed, and fixed :) Thanks!